### ISSUE TITLE
Increased gyration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9713]
**Preferred term label:**
Increased gyration

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

614207 
PMID: 23561846
2/3
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Is this different from polymicrogyria?

**Comment by pnrobinson** [comment]
There is not enough detail in the original publication to define this phenotype. I am closing this issue for now.
